src/codegeneration/ParenTrait.js

```diff
@@ -129,5 +130,16 @@ export function ParenTrait(ParseTreeTransformerClass) {
       }
       return new ArrayLiteralExpression(tree.location, elements);
     }
+
+    transformPropertyNameAssignment(tree) {
+      let name = this.transformAny(tree.name);
+      let value = this.transformAny(tree.value);
+      if (value.type === COMMA_EXPRESSION) {
```

Comment:
The arrow function gets transformed into a comma expression. This trait fixes that  so that individual transformers do not have to know about where the arrow is located in the tree.